docs/tutorials/getting-started-postgresql.md

```diff
@@ -23,10 +23,21 @@ packages:
     engine: "postgresql"
     schema: "schema.sql"
     queries: "query.sql"
+
+version: "2"
+sql:
+  - engine: "postgresql"
+    queries: "query.sql"
+    schema: "schema.sql"
+    gen:
+      go:
+        package: "tutorial"
+        out: "tutorial"
```

Comment:
It's confusing to have both version 1 and version 2 in the same YAML block. Since this is a getting started guide, let's only use version 2

Reply:
Whoops! I meant to remove version 1. Fixed!